corehq/apps/reports/static/reports/ko/saved_reports.js

```diff
@@ -148,6 +148,9 @@ var ReportConfigsViewModel = function (options) {
         $.ajax({
             type: "DELETE",
             url: options.saveUrl + '/' + config._id(),
+            beforeSend: function(xhr, settings) {
```

Comment:
seems like we'd have to do this for every `$.ajax` call (including `$.post`, etc), which seems a little annoying / non-DRY. Is there a way we can add this to the global `jQuery` post config?